torchtune/models/mistral/_model_builders.py

```diff
@@ -3,15 +3,13 @@
 #
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
-from typing import List, Optional
-from functools import partial
-
-from torch import nn
+from typing import List
```

Comment:
Please remove the docstring line in L71 about max_batch_size while you're in here